cmd/osbuild-worker/main.go

```diff
@@ -123,6 +123,9 @@ func RequestAndRunJob(client *worker.Client, acceptedJobTypes []string, jobImpls
 
 func main() {
 	var config struct {
+		Composer *struct {
+			Proxy string `toml:"proxy"`
```

Comment:
Feels like we should configure the url here too (separately / in a follow-up). I'm not sure configuring that on the command line makes any sense, and having the proxy in the config file draws attention to it, IMHO.

Reply:
Indeed, in the beginning, we just used the composer's URL from the systemd template unit with hardcoded path to the certificates but as we expanded the functionality of the worker, at one point we added a proper config. So I guess the correct way forward is to switch to a config completely, something for a follow-up I think.

Reply:
Agreed.